requirements-hw.txt

```diff
@@ -1,11 +1,11 @@
-speechRecognition==3.6.5
-pocketsphinx==0.1.3
+snowboy
```

Comment:
@hongquan , has your PR been merged in the snowboy repo??

Reply:
@stealthanthrax Not yet, but we tell `pip` to install `snowboy` from the wheel package we uploaded to https://repo.fury.io/fossasia/.